RF24Gateway.cpp

```diff
@@ -684,21 +703,24 @@ void RF24Gateway::handleTX()
     rxQueue.pop();
 }
 
-/***************************************************************************************/
+/***************************************************************************************
 
-void printPayload(std::string buffer, std::string debugMsg = "")
+template<class mesh_t, class network_t, class radio_t>
+void ESBGateway<mesh_t, network_t, radio_t>::printPayload(std::string buffer, std::string debugMsg)
 {
 }
 
-/***************************************************************************************/
+/***************************************************************************************
 
-void printPayload(char* buffer, int nread, std::string debugMsg = "")
+template<class mesh_t, class network_t, class radio_t>
+void ESBGateway<mesh_t, network_t, radio_t>::printPayload(char* buffer, int nread, std::string debugMsg)
 {
 }
 
 /***************************************************************************************/
```

Comment:
I did update these "definitions" to be used by the template class, but since they weren't technically scoped to `RF24Gateway`, I decided to comment out this entire section.